### DISCUSSION TITLE
Ceno Browser issue: Ouicrawled websites

### DISCUSSION THREAD

**Discussion by abhiXII** [sprblm/The-Design-We-Open#8] [Ideas]
Add a set of Ouicrawled websites on Ceno homepage
https://gitlab.com/censorship-no/ceno-browser/-/issues/104
Ceno crawls and creates a backup of a few websites that risk being blocked or censored in some regions so that users can access that content.

**Comment by Switchboards** [comment]
That’s a really helpful site, how much does it cost?

[cleantechcontrols.com.au](url)
